### PR TITLE
feat: use ArithmeticPatch as name

### PR DESCRIPTION
**Pull request by oleneyl**

### CODE REVIEW

File: simaple/spec/patch.py
Comment (icepeng) [inline simaple/spec/patch.py:1]: `return eval(value, global_variables) # pylint: disable=W0123`
issue: implementation does not match it's name.

**Review by icepeng — CHANGES_REQUESTED** [review]

File: simaple/spec/patch.py
Comment (oleneyl) [reply simaple/spec/patch.py:1]: 9f32be37cf350e0c9140ff4f35f402cd167eecdc
8177cda0a58730b7d6bc436628a1befa556a3c74

**Review by icepeng — APPROVED** [review]
less magic

File: simaple/spec/_math.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    ?expr: expr "+" term   -> add
+         | expr "-" term   -> sub
+         | term
+
+    ?term: term "*" factor -> mul
+         | term "/" factor -> div
+         | term "//" factor -> int_div
+         | term ">" factor  -> gt
+         | term "<" factor -> lt
```
Comment (icepeng) [inline simaple/spec/_math.py:17]: question: gt, lt가 필요한 상황이 있나요?

Comment (oleneyl) [reply simaple/spec/_math.py:17]: skill level이 1이상일때만 적용되어야 하는 경우에는 써야하더라고요

**Pull request merged**